frontend/src/pulldasher.tsx

```diff
@@ -1,39 +1,51 @@
 import usePullsState from './pulls-state';
 import PullsContext from './pulls-context';
+import { ChakraProvider } from "@chakra-ui/react"
 import { Pull } from './pull';
 import Navbar from './navbar';
 import { Column } from './column';
-import Container from 'react-bootstrap/Container';
-import Row from 'react-bootstrap/Row';
-import Col from 'react-bootstrap/Col';
+import { Box, SimpleGrid } from "@chakra-ui/react"
+import { theme } from './theme';
 
 export default function Pulldasher() {
    const pulls: Pull[] = usePullsState();
+   const pullsCIBlocked = pulls.filter(pull => pull.isCiBlocked());
+   const pullsDeployBlocked = pulls.filter(pull => pull.isDeployBlocked());
+   const pullsReady = pulls.filter(pull => pull.isReady());
    const pullsDevBlocked = pulls.filter(pull => pull.isDevBlocked());
    const pullsNeedingCR = pulls.filter(pull => !pull.isCrDone());
-   return (<PullsContext.Provider value={{pulls:pulls}}>
-      <Navbar/>
-      <Container>
-         <Row>
-            <Col>QA Leaderboard</Col>
-            <Col>CR Leaderboard</Col>
-         </Row>
-         <Row>
-            <Col>CI Blocked</Col>
-            <Col>Deploy Blocked</Col>
-            <Col>Ready</Col>
-         </Row>
-         <Row>
-            <Col>
-               <Column title={`Dev Block ${pullsDevBlocked.length}`}
-                  pulls={pullsDevBlocked}/>
-            </Col>
-            <Col>
-               <Column title={`CR ${pullsNeedingCR.length}`}
-                  pulls={pullsNeedingCR}/>
-            </Col>
-            <Col>QA</Col>
-         </Row>
-      </Container>
-   </PullsContext.Provider>);
+   const pullsNeedingQA = pulls.filter(pull => !pull.isQaDone());
+   return (<ChakraProvider theme={theme}>
+      <PullsContext.Provider value={{pulls:pulls}}>
+         <Navbar mb={4}/>
+         <Box maxW={1024} m="auto">
+         <SimpleGrid columns={3} spacing={6}>
```

Comment:
Nice use of SimpleGrid. I'm curious why there are 3 columns here if we only have 2 leaderboards